### PR TITLE
Adding periodic job for contrib

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

### CODE REVIEW

File: ci-operator/config/openshift/knative-eventing-contrib/openshift-knative-eventing-contrib-release-v0.18.0__46.yaml [hunk 1]
```diff
@@ -127,6 +127,11 @@ tests:
   commands: make test-e2e
   openshift_installer_src:
     cluster_profile: aws
+- as: e2e-aws-ocp-45-continuous
```
Comment (mgencur) [inline ci-operator/config/openshift/knative-eventing-contrib/openshift-knative-eventing-contrib-release-v0.18.0__46.yaml:130]: Should this be  e2e-aws-ocp-46-continuous ?

File: ci-operator/config/openshift/knative-eventing-contrib/openshift-knative-eventing-contrib-release-v0.17.1__46.yaml [hunk 2]
```diff
@@ -151,6 +151,11 @@ tests:
   commands: make test-e2e
   openshift_installer_src:
     cluster_profile: aws
+- as: e2e-aws-ocp-45-continuous
```
Comment (mgencur) [inline ci-operator/config/openshift/knative-eventing-contrib/openshift-knative-eventing-contrib-release-v0.17.1__46.yaml:154]: Should this be e2e-aws-ocp-46-continuous ?

File: ci-operator/config/openshift/knative-eventing-contrib/openshift-knative-eventing-contrib-release-v0.18.0__46.yaml (hunk 1 above)
Comment (matzew) [reply ci-operator/config/openshift/knative-eventing-contrib/openshift-knative-eventing-contrib-release-v0.18.0__46.yaml:130]: :stuck_out_tongue:

**Pull request merged**